src/iotjs_binding_helper.cpp

```diff
@@ -0,0 +1,99 @@
+/* Copyright 2016 Samsung Electronics Co., Ltd.
```

Comment:
As this code is originating from existing sources, better keep 2015-2016 copyright years